DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ScheduleCell.java

```diff
@@ -105,7 +109,15 @@ public void updateItem(Session item, boolean empty) {
         if (item != null && !empty) {
             updateListTile();
             secondaryGraphic.updateGraphic(session);
-            setGraphic(listTile);
+            if (showSessionType && item.isShowSessionType()) {
+                final BorderPane borderPane = new BorderPane(listTile);
```

Comment:
I'm worried about the performance impact of creating these nodes inside `updateItem`. I'd rather move BorderPane to the constructor and have all the cells use it (all with the listTile added to the center). I'll also create the HBox on top of the pane, and only play with its visible and managed properties inside this if.

Reply:
Good catch!